rocketpy/stochastic/stochastic_motor_model.py

```diff
@@ -14,5 +14,6 @@ class is used to standardize the input of the motor stochastic model.
 
     def __init__(self, object, **kwargs):
         self._validate_1d_array_like("thrust_source", kwargs.get("thrust_source"))
+        # TODO: this is non-sense, nobody will ever vary grain_number
```

Comment:
I believe we whould not leave this option here as it may conduct the user to a erratic behavior.

Varying the number of grains is not just something "unusual", it is rather something almost "impossible". How can you have an uncertainty on a measure like that?

In my opinion the number of grains and number of fins are two parameters that should be fixed in the monte carlo simulations.
Just like the `datum` reference and the `timezone`.

If you disagree with this position, please let me know.